src/components/input/file-input/types.ts

```diff
@@ -0,0 +1,6 @@
+import { MutableRefObject } from 'react';
+
+export interface FileInputProps {
+  inputRef: MutableRefObject<any>;
+  onChange: (e: any) => void;
```

Comment:
Think we can specify this as
```suggestion
  onChange: (e: React.FormEvent<HTMLInputElement>) => void;
```